integrations/README.md

```diff
@@ -0,0 +1,71 @@
+# Pocket Network 1.0 Cross Module Integrations
+
+<table style="text-align: center;">
+<tr style="text-align: center;">
+  <td style="border-collapse: collapse;">
+    <p align="center">
+        Alessandro De Blasis
+        @deblasis<br>
+    </p>
+  </td>
+
+  <td style="border-width: 0px;">
+    <p align="center">
+        Andrew Nguyen
+        @andrewnguyen22 <br>
+    </p>
+  </td>
+
+  <td style="border-width: 0px;">
+    <p align="center">
+        Daniel Olshansky
+        @olshansky <br>
+    </p>
+  </td>
+
+</tr>
+</table>
+
+<tr>
+<td style="border-width: 0px;">
+    <p align="center">
+        Version 1.0.0<br>
+    </p>
+</td>
+</tr>
+
+## Overview
+
+In this section, we will describe in which way the various modules interact with each other.
+
+## Introduction
+
+While we proceed in the R&D of the individual modules, we need to consider that even if we are developing them in a microservice fashion, they still need to communicate with each other so that the application and therefore the Protocol itself can function as expected.
+
+The purpose of this document is not to highlight the capabilities of each model but to focus on the point of contact/interactions with a brief explanation about the motivation behind the integration under discussion.
+
+## Integrations
+
+### Bus
+
+An application `Bus` serves as a cross-cutting integration medium.
+Without going too much into technical details, its main features for the purposes of this document are:
+
+- acts as a service discovery layer, allowing to get instances of a module from within another
```

Comment:
I see your points but perhaps this document should be less "loaded" with technicalities and try to explain things in simpler terms, good enough to have an idea of how it works at a high level

Regarding the diagram, if we copy-pasta it, we would have to keep track of it and keep them aligned.

I am sure that whoever reads this and is interested in how it works under the hood, they already pulled the main repo, probably as a first step actually.

Lemme know where your mind is at. Maybe we could ask Jess' point of view on this. I guess her input would be valuable. (not tagging her on purpose in the meantime).

My 2c. Let's keep it simple, actually, I was thinking that it might sound even too technical already perhaps 🤔

Reply:
I agree. This whole thing spec needs a revamp and I personally think https://github.com/pokt-network/pocket/issues/438 does a better job at introducing the P2P module. 

There will be more changes as we also have decided to adopt LibP2P.

- I've done an initial "revamp" of persistence (but will need another one in the future)
- Utility is in progress
- @bryanchriswhite should be able to take this one on over time
- Looking at @gokutheengineer for consensus

After we launch V1 (and have real data), we can write a single whitepaper capturing everything and publish on arxiv :)